cypress/integration/collections_spec.js

```diff
@@ -5,6 +5,10 @@ describe('Dashboard Collections Page', () => {
     it('should redirect to login page', () => {
       cy.visit('/#/collections');
       shouldBeRedirectedToLogin();
+
+      const name = 'MOD09GQ';
+      const version = '006';
+      cy.visit(`/#/collections/collection/${name}/${version}`);
```

Comment:
Did you mean to add `shouldBeRedirectedToLogin` after this line?

Reply:
Yes.  It's for the acceptance criteria.